webClient/src/app/core/menu-bar/menu-bar.component.ts

```diff
@@ -285,9 +303,9 @@ export class MenuBarComponent implements OnInit, OnDestroy {
     if (this.editorControl._isTestLangMode) {
       this.log.info(`Adding test language menu`);
       this.languagesMenu['TEST_LANGUAGE'] = TEST_LANGUAGE_MENU;
-    }
+    }       
 
-    this.subscription.add(this.appKeyboard.keyupEvent
+    this.keyBindingSub.add(this.appKeyboard.keydownEvent
       .filter(value => value.altKey).subscribe((event) => {
```

Comment:
`filter` should be removed here because you do filtering at https://github.com/zowe/zlux-editor/blob/795546b36110362ef206ebe128aa420b3a21177f/webClient/src/app/shared/editor-keybinding.service.ts#L39